golden_response/staging/get_pop_obs/moscow.json

```diff
@@ -2018,6 +1942,82 @@
         }
       ]
     },
+    "dc/p/89rd4chpd74r5": {
```

Comment:
This comes from REGION_DEMOGR_life_expectancy_and_mortality.csv (different  OECD dataset than above).

![image](https://user-images.githubusercontent.com/18621425/90837539-2e57be80-e307-11ea-94f6-5efb85685c2e.png)

And the value is attached to Moscow appropriately.
<img width="1226" alt="Screen Shot 2020-08-20 at 5 06 35 PM" src="https://user-images.githubusercontent.com/18621425/90837676-95757300-e307-11ea-8e40-96f4fa52330d.png">

Reply:
I am happy to change this to other place if moscow is not a representatively one.

Reply:
I don't think Moscow is issue here; we had some OECD place resolution process changes, so we do expect popobs to change. However, I thought that popobs DCID was deterministically computed, but the same PVs seem to have created different DCID. Maybe I am wrong.

Reply:
My usual question: I don't if its humanly reasonable to look through the proto diff of a 25K file and expect to catch bugs. Should we pick a simpler place? Prune the data?

Reply:
The order in this file is deterministic now. If we think we want to check new pops, it should be shown as trunk of diff. Otherwise I am totally okay to check the diff for a few selected pop id in the pop obs.

Reply:
Also to answer my own question for record-keeping: the DCID change was bc of [Years 0 14] becoming [Years - 14], which was hard to spot.